src/TheThingsNetwork.cpp

```diff
@@ -945,7 +945,7 @@ bool TheThingsNetwork::sendPayload(uint8_t mode, uint8_t port, uint8_t *payload,
 
 void TheThingsNetwork::sleep(uint32_t mseconds)
 {
-  if (mseconds < 100 || mseconds >= 4294967296)
```

Comment:
Actually this check doesn't make sense. An unsigned 32 bit parameter can never be >=2^32. So doing this check doesn't have any effect.